examples/SampleApp/src/screens/NewDirectMessagingScreen.tsx

```diff
@@ -163,7 +161,8 @@ export const NewDirectMessagingScreen: React.FC<NewDirectMessagingScreenProps> =
 
   const [focusOnMessageInput, setFocusOnMessageInput] = useState(false);
   const [focusOnSearchInput, setFocusOnSearchInput] = useState(true);
-  const [messageInputText, setMessageInputText] = useState('');
+  // As we dont use the first state variable we can simple not have it here and just separate with a comma within the array. Fix for no-usused-vars
+  const [, setMessageInputText] = useState('');
```

Comment:
```suggestion
  // As we don't use the state value, we can omit it here and separate it with a comma within the array.
  const [, setMessageInputText] = useState('');
```

Nifty, never think of/remember that this syntax is valid, neat trick. :) I added a suggestion for the wording of the comment, feel free to take parts or drop the whole thing. 

In the context of this being a sample makes this nice to have of course, but if it were in a production app it would be funny to have state being set and just never used, haha :D Good decision not to drop it entirely :)